autonomy/cli/deploy.py

```diff
@@ -90,7 +97,7 @@ def build_group() -> None:
 )
 @click.option(
     "--packages-dir",
-    type=click.Path(exists=True, dir_okay=True),
+    type=click.Path(dir_okay=True),
```

Comment:
here you effectively set it to false (as we have for keys here)
https://github.com/valory-xyz/open-autonomy/blob/9cb07231bad4044c0f70f5950c95031650104ec4/autonomy/cli/deploy.py#L70-L77

but for skaffold dir we do allow it. Is there a reason or shall we adopt the same pattern for both packages dir and skaffold dir?
https://github.com/valory-xyz/open-autonomy/blob/9cb07231bad4044c0f70f5950c95031650104ec4/autonomy/cli/deploy.py#L239-L243

Reply:
Keys file is required whereas the skaffold is optional, we can assume the skaffold dir will be the part of the build dir and define a default but cannot assume the same the keys file.